src/modules/clusters/cli/cluster.transformer.tsx

```diff
@@ -1,5 +1,6 @@
 import TimeAgo from 'javascript-time-ago';
 import en from 'javascript-time-ago/locale/en';
+import {CustomLogger} from "@cli/shared/services/logger.service";
```

Comment:
If you're using webstorm, add extension to support .editorconfig and also enable eslint --fix on save. 
For example, this import should have spaces before and after class name

Reply:
V